### ISSUE TITLE
biothing_type value should be a single value when it's a single-value list

### ISSUE THREAD

**Issue by newgene** [biothings/biothings.api#272]
When `ES_INDEX` contains multiple indices (e.g. with widecard), usually the `biothing_type` value in `/metadata` endpoint will be a list with the type values from all of these indices.

Here is an example from mygeneset.info API metadata, http://mygeneset.info/metadata:

```
{
  "biothing_type": ["geneset"],
...
}
```
The corresponding `ES_INDEX` setting is [here](https://github.com/biothings/mygeneset.info/blob/e14100ed5b4f52cfea6b7b1a02fda0b92da0eb58/src/config_web.py#L13):
```
ES_INDEX = 'mygeneset_current*'
```

We should convert the single value list to just a single value in this case.

**Comment by newgene** [comment]
See the issue with the current `biothing_type` value: https://github.com/biothings/biothings_client.py/issues/28

**Comment by newgene** [comment]
released in [0.12.0](https://github.com/biothings/biothings.api/releases/tag/v0.12.0)
